### PR TITLE
sorted lorem text.

### PR DESCRIPTION
**Pull request by cwiftdev**

### CODE REVIEW

File: index [hunk 1]
```diff
@@ -1 +1 @@
-Lorem impsun dolor sit amet.
```
Comment (cemaltysz) [inline index:1]: Lorem impsun to lorem impsun dolor... | please

Comment (cwiftdev) [reply index:1]: sure.

**Review by cemaltysz — CHANGES_REQUESTED** [review]
please, convert "Lorem impsun" to  "Lorem impsun dolor..."

**Review by cemaltysz — APPROVED** [review]
well done!

**Pull request merged**